src/tests/functional/plugin/cpu/single_layer_tests/grn.cpp

```diff
@@ -0,0 +1,125 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+
+using namespace CPUTestUtils;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+
+using GRNCPUTestParams = typename std::tuple<
+        InferenceEngine::Precision,
+        InferenceEngine::Precision,    // Input precision
+        InferenceEngine::Precision,    // Output precision
+        InferenceEngine::Layout,       // Input layout
+        InferenceEngine::Layout,       // Output layout
+        InputShape,                    // Input shape
+        float,                         // Bias
+        std::string>;                  // Device name
+
+class GRNLayerCPUTest : public testing::WithParamInterface<GRNCPUTestParams>,
+                        virtual public SubgraphBaseTest, public CPUTestsBase {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<GRNCPUTestParams> obj) {
+        InferenceEngine::Precision netPrecision;
+        InferenceEngine::Precision inPrc, outPrc;
+        InferenceEngine::Layout inLayout, outLayout;
+        InputShape inputShape;
+        float bias;
+        std::string targetDevice;
+
+        std::tie(netPrecision, inPrc, outPrc,
+                 inLayout, outLayout,
+                 inputShape,
+                 bias,
+                 targetDevice) = obj.param;
+
+        std::ostringstream result;
+        result << "IS=" << CommonTestUtils::partialShape2str({inputShape.first}) << "_";
+        result << "TS=";
+        for (const auto& item : inputShape.second) {
+            result << CommonTestUtils::vec2str(item) << "_";
+        }
+        result << "netPRC=" << netPrecision.name() << "_";
+        result << "inPRC=" << inPrc.name() << "_";
+        result << "outPRC=" << outPrc.name() << "_";
+        result << "inL=" << inLayout << "_";
+        result << "outL=" << outLayout << "_";
+        result << "bias="   << bias << "_";
+        result << "trgDev=" << targetDevice;
+
+        return result.str();
+    }
+
+protected:
+    void SetUp() override {
+        InferenceEngine::Precision netPrecision;
```

Comment:
Could we use ElementType instead of InferenceEngine::Precision?

Reply:
done